inference-engine/cmake/features_ie.cmake

```diff
@@ -109,3 +109,5 @@ ie_option(ENABLE_CLANG_FORMAT "Enable clang-format checks during the build" ON)
 set(IE_EXTRA_PLUGINS "" CACHE STRING "Extra paths for plugins to include into DLDT build tree")
 
 ie_dependent_option(ENABLE_TBB_RELEASE_ONLY "Only Release TBB libraries are linked to the Inference Engine binaries" ON "THREADING MATCHES TBB;LINUX" OFF)
+
+ie_option (USE_SYSTEM_PUGIXML "use the system copy of pugixml" OFF)
```

Comment:
all options in this file are named as `ENABLE_*`, don't you think we can use `ENABLE_SYSTEM_PUGIXML`?

Reply:
I'm not opposed to renaming it to `ENABLE_*`. I think there are two minor reasons not to do so:
* "enable" sends the wrong signal. We don't enable pugixml, we use a different one.
* Consistency with `NGRAPH_USE_SYSTEM_PROTOBUF`. If changing it to "enable" here, should I also change it there?

Reply:
@vinograd47  @ilyachur how do you think? `USE_SYSTEM_PUGIXML` looks good, but not consistent..

Reply:
I agree that it is not consistent.
All nGraph options are consistent because all options have a prefix `NGRAPH_`.
But I also agree that `USE` word is better for it.
I am ok with `USE_SYSTEM_PUGIXML`